doc/specs/#2325 - Default Profile Settings.md

```diff
@@ -0,0 +1,314 @@
+---
+author: Mike Griese @zadjii-msft
+created on: 2019-11-13
+last updated: 2019-11-13
+issue id: #2325
+---
+
+# Default Profile Settings
+
+## Abstract
+
+Oftentimes, users have some common settings that they'd like applied to all of
+their profiles, without needing to manually edit the settings of each of them.
+This doc will cover some of the many proposals on how to expose that
+functionality to the user in our JSON settings model. In this first draft, we'll
+examine a number of proposed solutions.
+
+## Inspiration
+
+During the course of the pull request review on [#3369], the original pull request for this feature's implementation, it became apparent that the entire team has differing opinions on how this feature should be exposed to the user. This doc is born from that discussion.
+
+## Solution Proposals

```

Comment:
At one point there was a 5th proposal:
- any of the settings for profiles should freely go in `globals`.
```json
{
    "defaultProfile": "<guid>",
    "fontFace": "Cascadia Code",
    "profiles": [ ... ]
}
```

From what I remember, this was abandoned because in a situation like the one above, it is unclear if "fontFace" would change the App's font vs the profiles' font.